### ISSUE TITLE
Allow users to make a donation to projects using ZOLD, not just CC

### ISSUE THREAD

**Issue by sereshqua** [zerocracy/farm#2027]
I thought that I would like to donate all my ZOLD earnings to some projects here. 

However, when I open an interface for donation ([see](https://www.0crat.com/contrib/C3NDPUA8L)), I am only able to pick up a payment from the list and use my CC to be able to donate.

But, I thought that it would be a great idea to allow project donation using ZOLD, so users would be able to connect to their wallets via 0crat and pick up any amount for donation.

**Comment by 0crat** [comment]
@g4s8[/z](https://www.0crat.com/u/g4s8) please, pay attention to this issue

**Comment by g4s8** [comment]
@yegor256 please look at this issue, I think it's a good idea, but we need to disable ZLD bonus for sandbox projects for ZLD donations.

**Comment by yegor256** [comment]
@g4s8 yes, I like the idea. And I agree that should disable bonuses first.

**Comment by 0crat** [comment]
Job [#2027](https://github.com/zerocracy/farm/issues/2027) is now in scope, role is `DEV`

**Comment by 0crat** [comment]
Bug was reported, see [§29](http://www.zerocracy.com/policy.html#29): +15 point(s) just awarded to @sereshqua[/z](https://www.0crat.com/u/sereshqua)
